<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>900</width><height>600</height></rect>
  </property>
  <property name="windowTitle">
   <string>Custom Language IDE</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="leftLayout">
      <item>
       <widget class="QLabel" name="codeLabel">
        <property name="text">
         <string>Code</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="codeTextEdit">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="runButton">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="rightLayout">
      <item>
       <widget class="QLabel" name="astLabel">
        <property name="text">
         <string>AST</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="astTree">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="headerHidden">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="outputLabel">
        <property name="text">
         <string>Output</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="outputTextEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <layoutdefault spacing="8" margin="10"/>
 </widget>
 <resources/>
 <connections/>
</ui>